backend/tracim_backend/lib/core/workspace.py

```diff
@@ -257,6 +256,25 @@ def get_all_for_user(
         query = query.order_by(Workspace.label)
         return query.all()
 
+    def get_all_accessible_for_user(self, user: User) -> typing.List[Workspace]:
+        """
+        Return workspaces accessible by user.
```

Comment:
by or for, maybe pick one. I'd say "by"

Reply:
I agree, but several other methods use `_for` so I preferred consistency over rightness.